lib/rex/post/local_fs.rb

```diff
@@ -0,0 +1,295 @@
+# -*- coding: binary -*-
+
+module Rex
+  module Post
+    ###
+    #
+    # This module provides commands for the local file system
+    #
+    ###
+    module LocalFs
```

Comment:
```suggestion
    module LocalFileSystem
```

Not a blocker, as I know it's consistent with meterpreter's `fs`